content/showcase/tracetronic/contents.lr

```diff
@@ -0,0 +1,8 @@
+name: TraceTronic
+---
+description:
+We wanted to redesign our website and chose Lektor because it combines the ease of use for regular content editors with the strenghts of static site generators. Also we like Python ;)
```

Comment:
Maybe you want `strengths` not `strenghts`? :smile:

Reply:
thanks :) i've fixed it